modules/using-aws-cli-create-iam-role-alb-controller.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using-aws-cli-create-iam-role-alb-controller_{context}"]
+= Using the AWS CLI to create an IAM role for the Controller
+
+.Prerequisites
+
+* You must have access to the `aws` command line interface.
+
+.Procedure
+
+. Generate a trusted policy file using your identity provider by running the following command:
```

Comment:
The same remarks about the trust policy and `<>` placeholders as for the operator.